pages/about/product_updates.md

```diff
@@ -13,21 +13,45 @@ eleventyNavigation:
 
 We’ll share notes about product releases and new features here.
 
+## January 31, 2023
+
+Today we launched a new way to get a .gov domain. You’re reading this on our new get.gov site. The purpose of this site is to help eligible government organizations get and manage .gov domains.
+
+### We made it easier to request a .gov domain
+
+Our old domain request process required a signature, typically from the senior elected or highest-ranking official in a government organization. This led to unnecessary delays for many organizations. **The new request process is fully digital**, doesn’t require a physical signature, and can usually be approved by someone with significant, executive responsibility in your organization, like a senior technology officer or chief administrative official.
+
+### We’re using Login.gov for authentication and identity verification
+
+Login.gov provides a simple and secure process for signing in to many government services with one account. We require you to verify your identity with Login.gov **before you can request your first .gov domain**. [Create your Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}. Read more about [verifying your identity with Login.gov](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/){.usa-link--external}. 
+
+### We improved this site to help you request and manage your .gov
```

Comment:
do we want to add "domain" to the end of this sentence? Like,

> We improved this site to help you request and manage your .gov domain

Reply:
I kind of like it without "domain." Do you think we should add it? @gabydisarli

Reply:
Hm, I lean towards adding it because we usually do say ".gov domain" right? I'll leave it up to you.